apps/src/templates/instructions/InlineFeedback.jsx

```diff
@@ -3,7 +3,7 @@ import Radium from 'radium';
 
 const InlineFeedback = ({ extra, message, styles }) => {
   return (
-    <div style={styles.container}>
+    <div style={styles.container} className="topInstructions-inline-feedback">
```

Comment:
I wonder if we can make it more clear that this is only used by UI tests? Some options I see
(1) Just add a comment
(2) change the name (something like `uitest-topInstructions-inline-feedback`)
(3) Use a data attribute instead, i.e. `data-inline-feedback={true}`

Thoughts?

The goal would be to make it clear that one doesnt need to worry about styles being applied via this classname.